sdk/lib/rtl/memstream.c

```diff
@@ -218,10 +219,14 @@ RtlReadOutOfProcessMemoryStream(
                                  Stream->Current,
                                  Buffer,
                                  CopyLength,
-                                 BytesRead);
+                                 &LocalBytesRead);
 
     if (NT_SUCCESS(Status))
-        Stream->Current = (PUCHAR)Stream->Current + *BytesRead;
+    {
+        Stream->Current = (PUCHAR)Stream->Current + LocalBytesRead;
+        if (BytesRead)
+            *BytesRead = (ULONG)LocalBytesRead;
```

Comment:
This cast is wrong. Either properly check for integer overflow, or don't cast so that MSVC's warning can help us catch this later.